mobile/src/reducers/feeds/feeds.epics.ts

```diff
@@ -0,0 +1,20 @@
+import { ActionsObservable, ofType } from 'redux-observable';
+import * as fAction from './feeds.action';
```

Comment:
and you need these `as` because...?

Reply:
Because you can't just write `import * from './feeds.action';`